app.js

```diff
@@ -0,0 +1,29 @@
+const express = require("express");
+const app = express();
+const { getAllTopics, getApi } = require("./controllers/news.controller");
+const { notFoundHandler } = require("./error_handling/errors");
+
+app.get("/api/topics", getAllTopics);
+
+app.get("/api", getApi);
+// CODE BELOW IS NOT WORKING
+
+app.get("/api/*", function (req, res) {
+  try {
+    throw new Error("Not found");
+  } catch (error) {
+    // Handle the error
+    res.status(404).send("Not found");
+  }
+});
```

Comment:
You don't need to catch anything here, what you are trying to do is matching any endpoints that go to /api/*, * meaning anything else that you haven't defined. 

You can then in this block essentially just do what you have on line 16, you will send a message of 'Not found' with a status of 404. I would check that this matches in your test 👍